.github/workflows/deploy-to-dotorg.yml

```
@@ -37,6 +37,6 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
         upload_url: ${{ github.event.release.upload_url }}
-        asset_path: ${{github.workspace}}/${{ github.event.repository.name }}.zip
-        asset_name: ${{ github.event.repository.name }}.zip
+        asset_path: ad-refresh-control/ad-refresh-control.zip
```

So not sure myself without being able to test this but I think the only part we need to change is the name of the zip file, as we customize the slug for this a few lines above (line 32). But I believe the path we'll still want to pull that dynamically:
```suggestion
        asset_path: ${{ github.workspace }}/ad-refresh-control.zip
```

---

Also see we're taking a different approach elsewhere, like Insecure Content Warning: https://github.com/10up/insecure-content-warning/blob/develop/.github/workflows/dotorg-push-deploy.yml#L30. That may be another approach to try if this one doesn't work